src/intTest/resources/failure_uat_data/NHS004/screening_interchange_parsing_failure_with_nhsack.edifact.dat

```diff
@@ -0,0 +1,43 @@
+UNA:+.? '
```

Comment:
I've noticed we've included `screening` in the test file names under `/NHS004/` but we've not done this for the `pathology` test files, is this needed?

Reply:
Our adaptor doesn't do anything with this segment, but the documentation says its required so I'll add it.

Reply:
I've added those segments in the recent push.